php/tests/PreviouslyGeneratedClassTest.php

```diff
@@ -12,7 +12,11 @@ class PreviouslyGeneratedClassTest extends TestBase
     public function testPrefixForReservedWords()
     {
         // In newer versions of PHP, we cannot reference the old class name.
-        if (version_compare(phpversion(), '8.1.0', '>=')) return;
+        if (version_compare(phpversion(), '8.1.0', '>=')) {
+          $this->markTestSkipped('PHP 8.1.0 and later do not allow reserved class name.');
+        };
```

Comment:
nit: remove extra semicolon
```suggestion
        }
```